solana/Cargo.toml

```diff
@@ -5,7 +5,11 @@ members = [
 ]
 
 [profile.release]
+opt-level = 3
 lto = true
+debug = false
+strip = "debuginfo"
+codegen-units = 1
```

Comment:
Brett added this... need to understand it better... is it needed ?

Reply:
"Link time optimization (LTO) is LLVM's way of implementing whole-program optimization. Cross-language LTO is a new feature in the Rust compiler that enables LLVM's link time optimization to be performed across a mixed C/C++/Rust codebase."

Looks like it would be a positive to have